rackcity/utils/asset_utils.py

```diff
@@ -54,18 +54,18 @@ def get_existing_power_port(port_name, asset_id, change_plan=None):
         return power_port
 
 
-def get_or_create_asset_cp(asset_id, change_plan):
+def does_asset_exist(asset_id, change_plan):
```

Comment:
since this function is only used once, i modified it to better suit its purpose

Reply:
might make more sense here to use `.exists()` on the query? since we don't actually want the object being returned? so 
```
return (AssetCP.objects.filter(id=asset_id, change_plan=change_plan.id).exists()) or (Asset.objects.filter(id=asset_id).exists())
```

Reply:
done